src/js/components/ConfigurationView.js

```diff
@@ -219,12 +305,17 @@ class ConfigurationView extends mixin(StoreMixin) {
       );
     }
 
+    let portsSection = this.getVIPPortDefinitionsSection(config);
+    if (!portsSection) {
+      portsSection = this.getPortDefinitionsSection(config);
+    }
```

Comment:
Why don't we just move this call (`this.getPortDefinitionsSection(config);`) to be returned in stead of `null` here: https://github.com/dcos/dcos-ui/pull/1057/files#diff-b56f4ccf31412430b3161e86081b2e8eR106